src/Symfony/Component/Console/Command/Command.php

```diff
@@ -227,7 +227,15 @@ public function run(InputInterface $input, OutputInterface $output)
 
         if (null !== $this->processTitle) {
             if (function_exists('cli_set_process_title')) {
-                cli_set_process_title($this->processTitle);
+                $success = @cli_set_process_title($this->processTitle);
+                if (!$success) {
+                    if ('Darwin' === PHP_OS) {
+                        $output->writeln('<comment>Running "cli_get_process_title" not as a superuser is not supported on OS X.</comment>');
+                    } else {
+                        $error = error_get_last();
+                        trigger_error($error['message'], E_USER_WARNING);
```

Comment:
Sure. But php forbids to use a `E_*` error type. Only `E_USER_*` are allowed.
Here I know it'll always be a `E_WARNING` as documented in the manual. So I just need to use `E_USER_WARNING`